src/slice.jl

```diff
@@ -121,13 +116,8 @@ end
 
 function slice_insert!(xs::Vector{T}, count::Integer, i::Integer, value::T) where T
     if count < length(xs) && 1 <= i <= count + 1
-        if isbits(T)
-            unsafe_copy!(pointer(xs, i+1),
-                         pointer(xs, i), count - i + 1)
-        else
-            for k in 0:(count - i)
-                @inbounds xs[count+1-k] = xs[count-k]
-            end
+        for k in 0:(count - i)
+            @inbounds xs[count+1-k] = xs[count-k]
```

Comment:
Same here, did the fast path not work in v0.7?  I think it really just needed a name change (which Compat would make work still in v0.6.2), from `unsafe_copy!` to `unsafe_copyto!`

Reply:
As per the previous comment.

Reply:
https://github.com/BioJulia/IntervalTrees.jl/pull/37/commits/bfe59b002e5f687d99faf5a65c1435efface8379 has the changes as suggested.